randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -588,12 +593,22 @@ Extra - room_id: [17]
   * Extra - room: 17
   * Extra - blockx: 44
   * Extra - blocky: 19
+  > Door to Lava Lake Annex
+      All of the following:
+          Morph Ball
+          Any of the following:
+              Gravity Suit and Varia Suit
```

Comment:
grav+varia is unnecesary, as you can already reach the door by going to upper door first and then free lava diving to the lower left

Reply:
That's true, will remove